### ISSUE TITLE
Ortholog Detection? 

### ISSUE THREAD

**Issue by Jome0169** [ComparativeGenomicsToolkit/hal#264]
Hi there,

I've been scouring the documentation for how --onlyOrthologs works in hal2maf but haven't been able to figure it out. I'm working on identifying conserved sequences in a series of distantly related grass species, so accurate identification of orthologs is quite important. Can you either point me to a resource or give me a simple run down of how orthologs are identified from the alignments? Would it be best to supplement progressive cactus with a gene syntenty approach as well?

Thanks so much for any information you can provide.

**Comment by glennhickey** [comment]
Yeah, it's pretty poorly documented.  `cactus-hal2maf` is a new wrapper for `hal2maf` that adds parallelization and a bunch of postprocessing.  Its documentation is freshly updated here: 

https://github.com/ComparativeGenomicsToolkit/cactus/blob/master/doc/progressive.md#maf

with the `--onlyOrhtologs` mode corresponding to the `--dupdeMode ancestral` option.
